src/main/MainMenu.cs

```diff
@@ -214,7 +226,42 @@ private void Import(RunLevel runLevel)
                 importer.Import();
             }
         }
+        /// <summary>
+        /// Export all machine-dependent information for Migration to new machine
+        /// </summary>
+        private void Migrate(RunLevel runLevel)
+        {
+            bool response = false;
+            bool encryptConfig = Properties.Settings.Default.EncryptConfig;
+            var settings = _container.Resolve<ISettingsService>();
+            if (runLevel != RunLevel.Unattended)
```

Comment:
Here we should check if the flag is present (.HasFlag) rather than doing a straight compare, because RunLevel may have multiple dimensions (e.g. Test, Force, etc.). You could also initialize the response variable with the flag check  to simplify the if's.